api/v1alpha1/bgppeer_webhook.go

```diff
@@ -41,6 +42,20 @@ func (bgpPeer *BGPPeer) SetupWebhookWithManager(mgr ctrl.Manager) error {
 
 //+kubebuilder:webhook:verbs=create;update,path=/validate-metallb-io-v1alpha1-bgppeer,mutating=false,failurePolicy=fail,groups=metallb.io,resources=bgppeers,versions=v1alpha1,name=bgppeervalidationwebhook.metallb.io,sideEffects=None,admissionReviewVersions=v1
 
+var _ webhook.Defaulter = &BGPPeer{}
+
+// Default implements webhook.Defaulter so a webhook will be registered for the type
+func (bgpPeer *BGPPeer) Default() {
+	bgppeerlog.Info("default", "name", bgpPeer.Name)
+
+	if bgpPeer.Spec.HoldTime == 0 {
```

Comment:
I think we don't need these anymore (on par with the other paramteres defaulted by metallb).

Reply:
well we need it as I explained in the commit description if user only configure Keepalivetime to value that is > than holdtime it will be too late to wait for metallb to validate this config webhook is better safety net here to catch this early and drop the config